dotcom-rendering/src/web/components/ShareIcons.tsx

```diff
@@ -67,6 +67,14 @@ const decideIconColor = (format: ArticleFormat, context: Context) => {
 			fill: ${palette.fill.shareIconGrayBackground};
 		`;
 	}
+	if (
+		format.design === ArticleDesign.NewsletterSignup &&
+		context === 'ArticleMeta'
+	) {
+		return css`
+			fill: ${neutral[7]};
+		`;
+	}
 	return css`
 		fill: ${palette.fill.shareIcon};
```

Comment:
Colour variations by format, we have a concept of `palette`. Have a look at `decidePalette`, which is where we centralise all colour decisions that depend on an article format.